tests/test_utility.py

```diff
@@ -430,6 +430,7 @@ def test_xf_fill_all():
         (_dax_check, {"d2": {"dtype": ["float64", int]}}),
         (_dax_check, {"no_dim": {"optional": True, "dtype": float}}),
         (_dax_check, {"d5": {"dtype": str}}),
+        (_dax_check, {"d5": {"dtype": [str]}}),
```

Comment:
can you check why this isn't working @mark141 ?

Reply:
The code did not check for subtypes of `str`, if `str` was in a list!